releasenotes/notes/add-image-pull-metrics-to-containerd-check-de6dcb700830df0c.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add support for image pull metrics in the containerd check.
+enhancements:
+  - |
+    Refactor ParseMetrics function into `pkg/util/prometheus` to parse response from the Prometheus endpoint.
```

Comment:
IMO you can remove this part from the changelog since it is not relevant for the end user (only useful for agent developers).

Reply:
Makes sense.

I removed it the enhancements section from the releasenote.

Thank you